clientd3d/d3drender.c

```diff
@@ -3371,18 +3372,21 @@ void D3DRenderNamesDraw3D(d3d_render_cache_system *pCacheSystem, d3d_render_pool
 				if (ROOM_OVERRIDE_DEPTH1 & GetRoomFlags())
 				{
 					depth = GetOverrideRoomDepth(SF_DEPTH1);
+               bUsingAlternateDepth = false;

```

Comment:
Are you sure that "false" is correct here, and in the other two cases below?  It is never set to true now.

Reply:
Just saw this, sorry for the delay. Fixed.